docs/podman-pod-stats.1.md

```diff
@@ -0,0 +1,92 @@
+% podman-p	qod-stats "1"
+
+## NAME
+podman\-pod\-stats - Display a live stream of resource usage statistics for the containers in one or more pods
+
+## SYNOPSIS
+**podman pod stats** [*options*] [*pod*]
+
+## DESCRIPTION
+Display a live stream of containers in one or more pods resource usage statistics
```

Comment:
Some lines in this page have ending periods, others do not.  It should be consistent and I'd vote for periods.

Reply:
Lets do that in a separate PR.